tests/apps/tower/conftest.py

```diff
@@ -29,19 +29,19 @@ class CaseIDs:
     COMPLETED: str = "cuddlyhen_completed"
 
 
-@pytest.fixture(name="tower_id")
+@pytest.fixture(name="tower_id", scope="session")
 def fixture_tower_id() -> str:
     """Return a NF Tower id."""
     return TOWER_ID
```

Comment:
These fixtures are cheap to setup, but it does not hurt to default to a session wide scope :)

Reply:
Yes and I also think it is good to start being explicit about this